rgdps/common/gd_obj.py

```diff
@@ -1,9 +1,8 @@
 from __future__ import annotations
 
-import base64
 from typing import Callable
-from typing import overload
 from typing import TypeVar
+from urllib.parse import quote
```

Comment:
nit but i would import urllib.parse here and use the fqdn when using it - "quote" is very generic, "urllib.parse.quote" is quite specific

Reply:
also might want to look into whether you want `quote` or `quote_plus` - you may need `quote_plus` if there are potentially `+` symbols (this is the case on osu servers for e.g. filenames)